src/ast/irbuilderbpf.cpp

```diff
@@ -301,7 +301,7 @@ CallInst *IRBuilderBPF::createMapLookup(const std::string &map_name,
                                         Value *key,
                                         const std::string &name)
 {
-  return createMapLookup(map_name, key, getInt8PtrTy(), name);
+  return createMapLookup(map_name, key, getPtrTy(), name);
```

Comment:
If `getPtrTy()` is available from LLVM 15, we'll need to add a wrapper as we support (i.e. run in CI) LLVM 10+.